docs/pages/access-controls/access-request-plugins/ssh-approval-opsgenie.mdx

```diff
@@ -0,0 +1,114 @@
+---
```

Comment:
You also need to add this page to [config.js](https://github.com/gravitational/teleport/blob/master/docs/config.json) so it appears in the menu on the left, and make sure to specify that it only applies to Cloud documentation scope (see [example here](https://github.com/gravitational/teleport/blob/master/docs/config.json#L54-L69)).